### ISSUE TITLE
Issue when run script pmhsc,py 

### ISSUE THREAD

**Issue by frusty-the-snowman** [vesche/pmhsc#1]
Can you help me establish why this is failing - 
This worked well with no issues 
python pmhsc.py --genmaps
The following had issues...

Python 3.9.5 (default, May 14 2021, 00:00:00)
Fedora 33 

Using all of these command yields the same issue.

python3 pmhsc.py --decrypt -i example/example.enc -o example/example.dec
python pmhsc.py --decrypt -i example/example.enc -o example/example.dec

placing file in the same directory 
python3 pmhsc.py --decrypt -i example.enc -o example.dec
python pmhsc.py --decrypt -i example.enc -o example.dec

`
Traceback (most recent call last):
  File "/home/kitty/Documents/codebreaker-master/pmhsc-master/pmhsc.py", line 151, in <module>
    main()
  File "/home/kitty/Documents/codebreaker-master/pmhsc-master/pmhsc.py", line 146, in main
    decrypt(args['mapping'], args['input'], args['output'])
  File "/home/kitty/Documents/codebreaker-master/pmhsc-master/pmhsc.py", line 96, in decrypt
    plaintext += search_mapping(l)
TypeError: can only concatenate str (not "NoneType") to str

`
What am I doing wrong ?
How can I fix this ?

**Comment by vesche** [comment]
Hi there!

Unfortunately, I can't reproduce the error. It seems to be working fine for me:
```
user@xps ~/c/pmhsc (master)> python --version
Python 3.9.5
user@xps ~/c/pmhsc (master)> python pmhsc.py --genmaps
Mapping completed in 13.80 seconds.
user@xps ~/c/pmhsc (master)> python pmhsc.py --encrypt -i example/example.txt -o example/example.enc
Encryption completed in 5.91 seconds.
user@xps ~/c/pmhsc (master)> python pmhsc.py --decrypt -i example/example.enc -o example/example.dec
Decryption completed in 153.16 seconds.
```

I do believe I know what caused your problem however. The `example/` files were generated using a `mapping.p` file (created doing the --genmaps command) that I generated on my own local machine. I used that mapping file to encrypt the file you are trying to decrypt. Our `mapping.p` files are different and you won't be able to decrypt the example file using your personally generated mappings. This is working as intended.

To get it to work follow the commands I did above. First encrypt the `example.txt` file and you will see that your encrypted file will be different from the original one. Now try to decrypt the file you encrypted :)

Also, I would recommend you check out rmhsc as it is better than pmhsc because it can encrypt any filetype while pmhsc can only do plaintext files.

Thanks for checking my project out! I hope this helps.
